docs/sdk/src/guides/your_first_node.rs

```diff
@@ -305,4 +307,30 @@ mod tests {
 			test_runtime_preset(FIRST_RUNTIME, 1000, preset);
 		});
 	}
+
+	#[test]
+	fn omni_chain_dev_mode_works() {
+		// omni-chain dev-mode works with dev_chain_spec
+		let dev_chain_spec = std::env::current_dir().unwrap().parent()
+		.unwrap().parent()
+		.unwrap().join("templates")
+		.join("parachain")
+		.join("dev_chain_spec.json");		
```

Comment:
nit: can we report the `dev_chain_spec.json` location to `CARGO_MANIFEST_DIR` instead? Just thinking this file hierarchy might change over time.

Reply:
this just leads doing the same as manifest will just point to the dir of cargo.toml of the crate and you will still need to go down 2 levels, so is just the same thing.

Reply:
My point was to not rely on the location of this file (meaning the `.parent()` calls). Feel it is qualitatively better, but no strong opinion.

Reply:
Ah, I think I get it know. You mean the manifest points to the crate's cargo.toml, not the workspace one. I think you are right. Please ignore this.